vite.config.ts

```diff
@@ -75,6 +90,7 @@ export default ({ mode }) => {
     server: {
       proxy: proxyConfig,
       port: 3000,
+      hmr,
```

Comment:
Typescript error:
Type 'string' is not assignable to type 'boolean | HmrOptions | undefined'